Dockerfile-nsv

```diff
@@ -34,11 +30,27 @@ ADD config/${PPM_CONFIG_FILE} /cvdi-stream/config/
 ADD data/${PPM_MAP_FILE} /cvdi-stream/config/
 ADD ./config /cvdi-stream/config
 
-# Do the build.
+# do the build
 RUN export LD_LIBRARY_PATH=/usr/local/lib && mkdir /cvdi-stream-build && cd /cvdi-stream-build && cmake /cvdi-stream && make
 
-# Add test data. This changes frequently so keep it low in the file.
+# === RUNTIME IMAGE ===
+FROM alpine:3.12
+USER root
+
+WORKDIR /cvdi-stream
+
+# add runtime dependencies
+RUN apk add --upgrade --no-cache \
```

Comment:
Is it possible to COPY the libs from build image?

Reply:
The dockerfiles have now been modified to include 3 image stages:
1. runtime-deps: Image with runtime dependencies
2. builder: Inherit from runtime-deps and build
3. final: Inherit from runtime-deps and copy built files

This removes the need to install the runtime dependencies twice.